src/Main.java

```diff
@@ -0,0 +1,17 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        List <Person> people = new ArrayList<>();
+
+        Person person1 = new Person("abc", "Marian", "Wisniewski", 34);
```

Comment:
Tworząc w taki sposób, dajesz możliwość wrzucenia jako identyfikatora dowolny obiekt. Nie jest to bezpieczne rozwiązanie. Pozbywasz się zalet które daje generyczność. Nie pilnuje Tobie typu, Czyli tak jakbyś ustawił identyfikator jako Object.

Zauważ, że w takim podejściu mógłbyś napisać 
Person person1 = new Person(2.22, "Marian", "Wisniewski", 34);
i też przejdzie...

Person person1;

person = new Person("abc", "Marian", "Wisniewski", 34);
person = new Person(2.33, "Marian", "Wisniewski", 34);

też przejdzie.

A chcielibyśmy tak:
Person<String> person1;

person = new Person("abc", "Marian", "Wisniewski", 34);
przejdzie...
person = new Person(2.33, "Marian", "Wisniewski", 34);
nie przejdzie...